README.md

```diff
@@ -79,10 +85,73 @@ const {
   positionals
 } = parseArgs({ args, options });
 console.log(values, positionals);
-// Prints: [Object: null prototype] { foo: true, bar: 'b' } []ss
+// Prints: [Object: null prototype] { foo: true, bar: 'b' } []
 ```
 
-`util.parseArgs` is experimental and behavior may change. Join the
+Detailed parse information is available for adding custom behaviours by specifying `tokens: true` in the configuration. The returned tokens have properties describing:
+
+* option:
+  * `kind`: {'option'}
```

Comment:
I stuck in the string, but see there is a [guideline](https://github.com/nodejs/node/blob/184af1c977271616e57588a6e4311c2095502394/doc/README.md):

> The type should refer to a Node.js type or a [JavaScript type](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Grammar_and_types#Data_structures_and_types).